cpp/util_module/algorithm/util.cpp

```diff
@@ -1,20 +1,48 @@
 #include "util.hpp"
 #include "md5.hpp"
 
-void Util::Md5(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory) {
+void Util::Md5Procedure(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory) {
   mgp::MemoryDispatcherGuard guard{memory};
   const auto arguments = mgp::List(args);
   const auto record_factory = mgp::RecordFactory(result);
   try {
-    const mgp::List arg_list = arguments[0].ValueList();
-    std::string return_string{""};
-    for (auto value : arg_list) {
-      return_string += value.ToString();
+    if (arguments[0].IsList()) {
+      const mgp::List arg_list = arguments[0].ValueList();
+      std::string return_string{""};
```

Comment:
Default ctor is empty ctor, so you can do it like this
```suggestion
      std::string return_string;
```